src/lib/modules/Patron/PatronBulkExtendLoans/PatronBulkExtendLoans.js

```diff
@@ -0,0 +1,54 @@
+import { Truncate } from '@components/Truncate';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button, Icon, Label, Popup } from 'semantic-ui-react';
+
+export default class PatronBulkExtendLoans extends Component {
+  render() {
+    const { patronPid, bulkLoanExtension, isLoading, ...props } = this.props;
+    return (
+      <Button
```

Comment:
Can we disable the button if there are no loans to extend? It might be hard...

Reply:
I was thinking about it but it seemed as little added value in comparison to the checks we would have to do:
1. check with loans are about to expire and if the extension limit has not been reached
2. for each loan check if the document is not overbooked
It would probably requite having small endpoint returning the effect of these checks... I could classify it as an improvement ? WDYT?